integration-tests/environment-helpers/use-db.js

```diff
@@ -172,18 +175,30 @@ module.exports = {
       const pgConnection = await pgConnectionLoader({ configModule, container })
       instance.setPgConnection(pgConnection)
 
-      const { runMigrations } = await medusaAppLoader(
-        { configModule, container },
-        { registerInContainer: false }
-      )
+      const migrationPromises = []
+      for (const [, options] of Object.entries(configModule.modules)) {
```

Comment:
@adrien2p, something like this for running module migrations before loaders?

cc @carlos-r-l-rodrigues

Reply:
What is the issue happening here? 
if we do that, we'll miss link-modules' migration.

Reply:
`MedusaApp` will run module loaders before migrations. I am creating default currencies and countries in the loader and this will fail with a "table doesn't exist" error.

Reply:
But I can see several suites are failing now. Let me rethink this approach and revert back.